### ISSUE TITLE
Error recording synapse properties for cells with multiple synapse types

### ISSUE THREAD

**Issue by tjbanks** [AllenInstitute/bmtk#68]
The most recent feature merge for recording synapse properties works great. Initial testing went fine until I had multiple synapse types on the post synaptic cell. I started running into this issue:

```
(base) C:\Users\Tyler\Desktop\git_stage\hippocampus-bmtk\HummosBanks-bmtk>python run_bionet.py
C:\Users\Tyler\Anaconda3\lib\site-packages\h5py\__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2019-03-19 17:57:25,900 [INFO] Created log file
2019-03-19 17:57:26,291 [INFO] Building cells.
2019-03-19 17:57:27,072 [INFO] Building recurrent connections
2019-03-19 17:57:36,431 [INFO] Building virtual cell stimulations for exc_spikes
2019-03-19 17:57:36,778 [INFO] Running simulation for 100.000 ms with the time step 0.100 ms
2019-03-19 17:57:36,778 [INFO] Starting timestep: 0 at t_sim: 0.000 ms
2019-03-19 17:57:36,778 [INFO] Block save every 1000 steps
Traceback (most recent call last):
  File "C:\Users\Tyler\Anaconda3\lib\site-packages\bmtk-0.0.7-py3.6.egg\bmtk\simulator\bionet\biosimulator.py", line 253, in post_fadvance
    mod.step(self, self.tstep)
  File "C:\Users\Tyler\Anaconda3\lib\site-packages\bmtk-0.0.7-py3.6.egg\bmtk\simulator\bionet\modules\record_netcons.py", line 129, in step
    self._var_recorder.record_cell(gid, var_name, syn_values, tstep)
  File "C:\Users\Tyler\Anaconda3\lib\site-packages\bmtk-0.0.7-py3.6.egg\bmtk\utils\io\cell_vars.py", line 205, in record_cell
    buffer_block[update_index, gid_beg:gid_end] = seg_vals
ValueError: cannot copy sequence with size 22 to array axis with dimension 25
NEURON: PyObject method call failed: post_fadvance
 near line 0
 {DA_IZH = 2}
             ^
        advance()
      step()
    continuerun(100)
  run(100)
Traceback (most recent call last):
  File "run_bionet.py", line 94, in <module>
    run('simulation_config.json')
  File "run_bionet.py", line 86, in run
    sim.run()
  File "C:\Users\Tyler\Anaconda3\lib\site-packages\bmtk-0.0.7-py3.6.egg\bmtk\simulator\bionet\biosimulator.py", line 224, in run
    h.run(h.tstop)        # <- runs simuation: works in parallel
RuntimeError: hoc error
```

It appears there's an array mismatch in cell_vars.py brought on when registering cells in record_netcons.py. 

I've got a followup pull request to fix it with your approval. (#69)

**Comment by kaeldai** [comment]
Merged. Thanks!
